src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_FindReferences_Legacy.cs

```diff
@@ -62,19 +59,33 @@ public static partial class SymbolFinder
         /// information as the search is undertaken.</param>
         /// <param name="documents">An optional set of documents to be searched. If documents is null, then that means "all documents".</param>
         /// <param name="cancellationToken">An optional cancellation token.</param>
-        public static async Task<IEnumerable<ReferencedSymbol>> FindReferencesAsync(
+        public static Task<IEnumerable<ReferencedSymbol>> FindReferencesAsync(
             ISymbol symbol,
             Solution solution,
             IFindReferencesProgress progress,
             IImmutableSet<Document> documents,
             CancellationToken cancellationToken = default)
+        {
+            return FindReferencesAsync(
+                symbol, solution, progress, documents,
+                FindReferencesSearchOptions.Default, cancellationToken);
+        }
+
+        internal static async Task<IEnumerable<ReferencedSymbol>> FindReferencesAsync(
```

Comment:
>FindReferencesAsync [](start = 66, length = 19)

I don't have full context, but I'm surprised to see we're adding a new API to the "legacy" set.
Why isn't the API which is added to the "current" set sufficient to fulfill scenarios?

Reply:
That's a good point.  We likely don't need to update hte legacy set.  i'll check and see if i can avoid that.  Thanks!